README.md

```diff
@@ -1,67 +1,98 @@
-# Godot-AdMob-Editor-Plugin
-This repository is for Godot's Addons to integrate natively AdMob to your Game Project without much configurations, with a beautiful UI and directly inside Godot Editor!
+[![Download Asset Library](https://img.shields.io/badge/Download-Asset%20Library-darkgreen?style=for-the-badge)](https://godotengine.org/asset-library/asset/1108)
+![Stars](https://img.shields.io/github/stars/Poing-Studios/godot-admob-android?style=for-the-badge)
+![License](https://img.shields.io/github/license/Poing-Studios/godot-admob-android?style=for-the-badge)
 
-<u>is pretty basic right now, but with time will be improved</u>
+# Godot AdMob Editor Plugin
 
-Tutorial Video: https://youtu.be/ZnlH3INcAGs
+This addon provides an easy and beautiful way to configure AdMob directly through the editor. Supports [godot-admob-android](https://github.com/Poing-Studios/godot-admob-android) and [godot-admob-ios](https://github.com/Poing-Studios/godot-admob-ios).
 
-# Installation:
-Please read this tutorial: https://docs.godotengine.org/pt_BR/stable/tutorials/plugins/editor/installing_plugins.html
+### [Installation and Usage Tutorial](https://www.youtube.com/watch?v=ZnlH3INcAGs)
 
-Manual:
-1. Go to Tags of the project: https://github.com/Poing-Studios/Godot-AdMob-Editor-Plugin/tags
-2. Download and extract the `.zip` or `.tar.gz`
-3. Put the "addons/admob" folder which was extracted into your "res://addons" of your Game Project
+# Installation
 
-Godot Asset Library:
-1. Find the Plugin `AdMob` then download and install
-2. Enable the Plugin: https://docs.godotengine.org/pt_BR/stable/tutorials/plugins/editor/installing_plugins.html#enabling-a-plugin
+Godot Asset Library (recommended):
+1. Find the AdMob plugin by `poing.studios` \
+   <img height=100 src="https://raw.githubusercontent.com/Poing-Studios/godot-admob-editor/main/addons/admob/assets/asset-library.png">
```

Comment:
It wouldn't work because that would point to `https://github.com/Poing-Studios/godot-admob-editor/addons/admob/assets/asset-library.png` which isn't a valid image.